File: src/main/java/um/tesoreria/core/model/view/ContratadoPersona.java
/**
 * 
 */
package um.tesoreria.core.model.view;

import java.io.Serializable;
import java.math.BigDecimal;

import jakarta.persistence.Entity;
import jakarta.persistence.Id;
import jakarta.persistence.Table;

import org.hibernate.annotations.Immutable;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

/**
 * @author daniel
 *
 */
@Data
@Entity
@Table(name = "vw_contratado_persona")
@Immutable
@NoArgsConstructor
@AllArgsConstructor
public class ContratadoPersona implements Serializable {
	/**
	* 
	*/
	private static final long serialVersionUID = -3947729039514048661L;

	@Id
	private Long contratadoId;

	private BigDecimal personaId;
	private Integer documentoId;
	private String apellido;
	private String nombre ;

}

File: src/main/java/um/tesoreria/core/kotlin/model/view/ContratadoPersona.kt
package um.tesoreria.core.kotlin.model.view

import jakarta.persistence.Entity
import jakarta.persistence.Id
import jakarta.persistence.Table
import org.hibernate.annotations.Immutable
import java.math.BigDecimal

@Entity
@Table(name = "vw_contratado_persona")
@Immutable
data class ContratadoPersona(

    @Id
    var uniqueId: String? = null,

    var personaId: BigDecimal? = null,
    var documentoId: Int? = null,
    var apellido: String? = null,
    var nombre: String? = null,
    var cuit: String? = null

)
